app/templates/merchant/manage_cart.html

```diff
@@ -1,5 +1,48 @@
 {% extends 'layouts/base.html' %}
 
+
+{% macro vendor_modal_table(vendor, cart_items) %}
+
+    <div class="ui list ">
+        <div class="item"><b>Company name: </b> {{  vendor.company_name }}</div>
+        <div class="item"><b>Vendor name: </b> {{  vendor.full_name() }}</div>
+        <div class="item"><b>Email: </b> {{  vendor.email }}</div>
+    </div>
+
+
+    <table class="ui compact table">
+        <thead>
+        <tr>
+            <th>Item name</th>
+            <th>Quantity</th>
+            <th>Unit Price</th>
+            <th>Total</th>
+        </tr>
+        </thead>
+        <tbody>
+        {% for cart_item in cart_items %}
+            <tr>
+                <td>{{ cart_item.listing.name }}</td>
+                <td>{{ cart_item.quantity| int() }} {{ cart_item.listing.category.unit }}</td>
+                <td>${{ '%0.2f' | format(cart_item.listing.price) }} / {{ cart_item.listing.category.unit }}</td>
```

Comment:
im seeing `'%0.2f' | format(...)` a lot. is there a way to just generalize it under a more concise method? (not really something to worry about, just something to consider)